packages/common/src/utils/object/common.ts

```diff
@@ -92,3 +92,7 @@ export const filterEntries = pickBy;
 
 export const mapValue = <T, R>(obj: Record<string | number | symbol, T>, callback: (value: T) => R) =>
   Object.fromEntries(Object.entries(obj).map(([key, value]) => [key, callback(value)]));
+
+export const shallowPartialEquals = (obj: unknown, partial: unknown) => {
```

Comment:
I don't know if we should be just operating on unknown values, this kind of makes this function less useful, if we add types we can make it more clear the relationship between `obj` and `partial`

```suggestion
export const shallowPartialEquals = <T extends object>(obj: T, partial: Partial<T>) => {
```